src/DynamicRupture/Output/Builders/ElementWiseBuilder.hpp

```diff
@@ -35,8 +35,9 @@ class ElementWiseBuilder : public ReceiverBasedOutputBuilder {
     const auto numFaultElements = meshReader->getFault().size();
     const auto numSubTriangles = faultRefiner->getNumSubTriangles();
 
-    logInfo(localRank) << "CPP: Initialising Fault output. "
-                       << "Number of sub-triangles: " << numSubTriangles;
+    logInfo(localRank) << "Initialising Fault output."
```

Comment:
This may be a more general question... British or American spelling?

Reply:
Changed to `Initializing`